symfony/src/Controller/NoticeController.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+namespace App\Controller;
+
+use App\Form\NoticeType;
+use App\Repository\NoticeRepository;
+use App\Service\LoggerService;
+use Doctrine\ORM\EntityManagerInterface;
+use Pmi\Audit\Log;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\Form\FormError;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Routing\Annotation\Route;
+
+/**
+ * @Route("/s/admin/notices")
+ */
+class NoticeController extends AbstractController
+{
+    /**
+     * @Route("/", name="admin_notices")
+     */
+    public function index(NoticeRepository $noticeRepository)
+    {
+        $notices = $noticeRepository->findBy([], ['id' => 'asc']);
+        return $this->render('notice/index.html.twig', [
+            'notices' => $notices,
+        ]);
+    }
+
+    /**
+     * @Route("/notice/{id}", name="admin_notice")
+     */
+    public function edit(NoticeRepository $noticeRepository, EntityManagerInterface $em, LoggerService $loggerService, Request $request, $id=null)
+    {
+        if ($id) {
+            $notice = $noticeRepository->find($id);
+            if (!$notice) {
+                throw $this->createNotFoundException('Page notice not found.');
+            }
+        } else {
+            $notice = null;
+        }
+
+        $form = $this->createForm(NoticeType::class, $notice, ['timezone' => $this->getUser()->getTimezone()]);
```

Comment:
This isn't working because the underlying `getTimezone` method is currently broken (and unused). We need to either get this from the info array or fix the User class to get the timezone directly.

Reply:
Should be resolved as part of 39524cb.